src/main/java/com/android/volley/NetworkResponse.java

```diff
@@ -48,11 +47,11 @@ public NetworkResponse(int statusCode, byte[] data, Map<String, String> headers,
     }
 
     public NetworkResponse(byte[] data) {
-        this(HttpStatus.SC_OK, data, Collections.<String, String>emptyMap(), false, 0);
+        this(HttpURLConnection.HTTP_OK, data, Collections.<String, String>emptyMap(), false, 0);
```

Comment:
I always thought this was against the style guide, but it doesn't look like it. Well, I guess I personally prefer having the verbosity since I feel like imports are generally not noticed when looking at a file and the source of these constants is kind of relevant, though I wouldn't push back if a static import were used elsewhere. Going to just leave this as is to avoid churn.